services/client/options.go

```diff
@@ -57,13 +55,6 @@ type Options interface {
 	// SetInitTimeout sets the InitTimeout
 	SetInitTimeout(t time.Duration) Options
 
-	// HeartbeatCheckInterval is the interval for heartbeat check
```

Comment:
because previously I call heartbeatStore.Get() every HeartbeatCheckInterval, now if waits for the notification on the watch to call Get()